library/src/main/java/com/google/android/exoplayer/extractor/webm/WebmExtractor.java

```diff
@@ -1303,6 +1314,35 @@ public void initializeOutput(ExtractorOutput output, int trackId, long durationU
       this.output.format(format);
     }
 
+    private static Pair<List<byte[]>, Long> parseFourCCPrivate(ParsableByteArray buffer)
+            throws ParserException {
+        try {
+            buffer.setPosition(buffer.getPosition()+16); // size, skip width(4), height(4),
+                                                         // planes(2), and bitcount(2)
+            long biCompression = buffer.readLittleEndianUnsignedInt();
+            buffer.setPosition(buffer.getPosition()+20); // skip sizeimage(4), xpel/meter(4),
+                                                       // ypel/meter(4), clrUsed(4), clrImportant(4)
+            List<byte[]> privateData = new ArrayList<>();
+            final int baseBytesLeft = buffer.bytesLeft();
+            byte[] scratchBytes = new byte[baseBytesLeft];
+            buffer.readBytes(scratchBytes, 0, baseBytesLeft);
+
+            int realStartPos;
+            for(realStartPos = 0; realStartPos < (baseBytesLeft - 4); ++realStartPos) {
```

Comment:
https://matroska.org/technical/specs/codecid/index.html
https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=2&ved=0ahUKEwi6ko7ikfjLAhXEjZQKHQQdAVwQFgghMAE&url=https%3A%2F%2Fhaali.su%2Fmkv%2Fcodecs.pdf&usg=AFQjCNHwS8o7g09E0qzFNfwPxkY9a_wlmQ&sig2=gbb7_rCon1r_oJWrUN48VA&bvm=bv.118443451,d.dGo&cad=rja

Search for V_MS/VFW/FOURCC (and vc1).  The remaining bytes are private data.  I believe the first byte is a reserve before Profile/PPS/SPS and if you pass the full private data in you get a pink screen which is not necessarily enjoyable viewing experience. ;)

This was on a Shield TV device.